docs/01_introduction/README.md

```diff
@@ -322,10 +323,10 @@ which is used and developed by various HPC centres & consortia worldwide, includ
 * [University of Melbourne, Australia](https://dashboard.hpc.unimelb.edu.au/)
 * [HPCNow!](https://hpcnow.com/)
 
-Today, an [experienced team of HPC experts](https://easybuild.readthedocs.io/en/latest/Maintainers.html) actively maintains the project,
+Today, an [experienced team of HPC experts](https://easybuild.readthedocs.io/en/latest/Maintainers.html) actively maintain the project,
```

Comment:
should be "maintains", since it's "team (of ...) actively maintains"?

Reply:
Team is a silly word. Or rather, collective nouns are silly. https://www.quora.com/Which-one-is-correct-‘My-team-are’-or-‘My-team-is’. In this context, I think "maintains" will read better for the vast majority of the readers (read: non-brits). That matters more than any form of attempted linguistic correctness. Reverted.